docs/tutorials/layering-multiple-environments.md

```diff
@@ -107,6 +107,16 @@ You can layer as many environments as you want.
 If two environments contain the same package,
 Flox will use the package from the last environment activated.
 
+You can use `flox envs` to see the environments you have activated.
+
+```bash
+flox envs
+✨ Active environments:
+  material-ui  /home/youruser/material-ui
+  default      /home/youruser
+
+Inactive environments:
+```
```

Comment:
Otherwise, CI will complain about the lack of whitespace around the backticks.
````suggestion
You can use `flox envs` to see the environments you have activated.

```bash
flox envs
✨ Active environments:
  material-ui  /home/youruser/material-ui
  default      /home/youruser

Inactive environments:
```

````